aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -319,8 +319,7 @@ impl AptosVM {
         log_context: &AdapterLogSchema,
         change_set_configs: &ChangeSetConfigs,
     ) -> Result<(VMStatus, TransactionOutputExt), VMStatus> {
-        let storage_with_changes =
-            DeltaStateView::new(resolver, user_txn_change_set_ext.write_set());
+        let storage_with_changes = DeltaStateView::new(resolver, &user_txn_change_set_ext);
         // TODO: at this point we know that delta application failed
```

Comment:
This comment seems very weird and outdated, we should probably fix it. @georgemitenkov can you look as well?
Do we know that the delta application failed?
Should we remove the part where it says the only use-case is for gas fees? (soon to not be the case)

Reply:
Yes its super-outdated. @vusirikala let's get rid of this comment.